TeamSPBackend/csv_data/csvRead.py

```diff
@@ -1,33 +1,33 @@
 import csv
-from django.utils.timezone import now
-from TeamSPBackend.team.models import Team
-filename = "template.csv"
-rows = []
-colNames = []
+import codecs
+from TeamSPBackend.team.models import Team, TeamMember
 
-with open (filename, 'r') as csvfile:
-    data = csv.reader(csvfile)
-    colNames = next(data)
-    for row in data:
-        rows.append(row)
 
-    for row in rows:
-        name = row[0]
-        description = row[1]
-        subject_code  = row[2]
-        project_name = row[3]
-        start_date = row[4]
-        expire_date = row[5]
-        supervisor = row[6]
-        student_ids = row[7]
-        list_students_ids = student_ids.split(",")
-        member_names = rows[8]
-        list_member_names = member_names.split(",")
-        member_emails = rows[8]
-        list_member_emails = member_emails.split(",")
+def read_csv(file, subject_id: int):
+    with file as csv_file:
+        rows = []
+        data = csv.reader(codecs.iterdecode(csv_file, 'utf-8'))
+        for row in data:
+            rows.append(row)
+            print(row)
+
+        for row in rows:
+            name = row[0]
+            description = row[1]
+            supervisor_id = row[2]
+            create_date = row[3]
+            expired = row[4]
```

Comment:
If we are not keeping 'expired' in the Team model, this needs to be removed from the csv reader.